app/script/auth/page/Verify.js

```diff
@@ -48,16 +56,16 @@ const Verify = ({account, authError, history, ...connected}) => {
           <Text data-uie-name="label-with-email">
             {'Enter the verification code we sent to'}
             <br />
-            {'email@mail.com'}
+            {account.email}
           </Text>
-          <CodeInput data-uie-name="enter-code" autoFocus style={{marginTop: 10}} onCompleteCode={createAccount} />
+          <CodeInput autoFocus style={{marginTop: 10}} onCompleteCode={createAccount} data-uie-name="enter-code" />
           <ErrorMessage>{authError}</ErrorMessage>
         </div>
         <div>
-          <Link data-uie-name="do-resend-code" href="#">
+          <Link onClick={resendCode} data-uie-name="do-resend-code">
             {'RESEND CODE'}
```

Comment:
Can we transform the link text to uppercase and stop screaming here?

Reply:
@AndyLnd is on it already